books/free-programming-books-langs.md

```diff
@@ -429,7 +429,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Beej's Guide to the GNU Debugger (GDB)](http://beej.us/guide/bggdb/) - Brian "Beej Jorgensen" Hall (HTML)
 * [Build Your Own Lisp](http://www.buildyourownlisp.com) - Daniel Holden
 * [C Elements of Style](http://www.oualline.com/books.free/style/) - Steve Oualline
-* [C for Python Programmers](http://www.cburch.com/books/cpy/) - Carl Burch
+* [C for Python Programmers](https://www.cs.toronto.edu/~patitsas/cs190/c_for_python.html) - Carl Burch
```

Comment:
What is the problem here? http://www.cburch.com/books/cpy/ is still available

Reply:
is return myself "Site not found" in dreamhost page

Reply:
even for the domain page

Reply:
works fine for me too

Reply:
I connect myself to the website with https by default, my bad..

Reply:
I going to close this pr since this is not a issue. Thank for your review.